docs/questions/query-builder/expressions/cumulative.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Cumulative count and sum
+redirect_from:
+  - /docs/latest/questions/expressions/cumulativesum
+  - /docs/latest/questions/expressions/cumulativecount
+---
+
+# Cumulative count and sum
+
+Cumulative count computes the additive total count of rows across a breakout. Cumulative sum computes the rolling sum (also known as running total) of a column across a breakout.
+
+## Syntax
+
+You can use the predefined "Cumulative count" and "Cumulative sum" metrics, or a custom expression:
+
+```
+CumulativeCount
+```
+
+```
+CumulativeSum(column)
+```
+
+Cumulative metrics can only be used in the query builder's **Summarize** step — you cannot use cumulative metrics to create a custom column or in a custom filter.
+
+## How cumulative metrics are computed
+
+Under the hood, cumulative metrics is computed in three steps:
+
+1. Group records using the breakout from the **Group by** block,
+2. Count the rows in each group,
+3. For each group, compute the cumulative total of the counts in all previous groups, including the current group.
+
+Because cumulative metrics use values from previous rows, the sort order in the breakout column matters.
+
+For example, if you have a table with data grouped by month, Metabase will compute the cumulative count like this:
+
+| Month    | `Count` | `CumulativeCount` |
+| -------- | ------- | ----------------- |
+| July     | 5       | 5                 |
+| November | 4       | 5+4 = 9           |
+| March    | 2       | 5 + 4 + 2 = 11    |
+
+If the sort is changed (while values remain the same), then the cumulative count changes as well:
+
+| Month    | `Count` | `CumulativeCount` |
+| -------- | ------- | ----------------- |
+| March    | 2       | 2                 |
+| July     | 5       | 2 + 5 = 7         |
+| November | 4       | 2 + 5 + 4 = 11    |
+
+Metabase will automatically sort the data by the breakout column in ascending order using the logic that your database uses for the column's data type. If you add a manual sort, it will only apply _after_ cumulative count or sum computes its results. Sorting here has no effect on how Metabase computes the cumulative metric.
+
+If you want to use a different order (for example, sort by the breakout column but in descending order), you can [use SQL](#sql).
+
+## Cumulative metrics with multiple breakouts
+
+Because cumulative metrics calculate its summaries based on previous rows, Metabase needs to determine what those previous rows are. It will always order the rows based on the first breakout. So if there are multiple breakout columns specified in **Group by**, Metabase will:
```

Comment:
```suggestion
Because cumulative metrics calculate its summaries based on previous rows, Metabase needs to determine what those previous rows are. Cumulative metrics will always order the rows based on the _first_ breakout. So if there are multiple breakout columns specified in **Group by**, Metabase will:
```